test/Issue.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+const sinon = require('sinon');
+require('chai').use(require('sinon-chai'));
+const Issue = require('../src/models/Issue');
+
+describe('Issue model', () => {
```

Comment:
@michaelmurray6298 these are great starting tests! Do you think you have time to test more than the fact that they exist?